ccron/include/ccron/ticks_generator.hpp

```diff
@@ -65,10 +70,14 @@ class TicksGenerator {
   const IPendingRequest *pending_req_{nullptr};
```

Comment:
not related to this PR, but must say that ```IPendingRequest``` is confusing name, I didn't expect the client manager to implement it

Reply:
It has been introduced for testing only, to avoid constructing a real clients manager in unit tests. It is not an interface users must rely on.

I am open to changing the name. Any ideas for a different one?

Reply:
not really :) any way it's nit